components/eam/cime_config/testdefs/testmods_dirs/eam/ne4_v3atm_rtmoff/user_nl_eam

```diff
@@ -0,0 +1,43 @@
+! The settings needed for chemUCI-Linozv3 in v3atm's 4th full smoke test
+! This is to ease testing with F20TR_chemUCI-Linozv3
+
+ ncdata = '$DIN_LOC_ROOT/atm/cam/inic/homme/20220906.v2.LR.bi-grid.amip.chemUCI_Linozv3.eam.i.2000-01-01-00000.mapped_to_ne4np4.nc'
+ tropopause_e90_thrd    = 80.0e-9
+ history_gaschmbudget_2D = .false.
```

Comment:
@wlin7 , these history flags are for chemistry budget diagnostics. Just double check if these tests are not intended to have these diagnostic output.